corehq/apps/userreports/reports/filters/values.py

```diff
@@ -223,15 +223,7 @@ def _is_empty(self):
         """
         Returns true if operand should be treated like an empty string.
```

Comment:
Nit: this docstring seems possibly misleading now that `_is_empty()` is a superset of `_is_null()`. Specifically the part that says "treated like an empty string". If I may treat it like an empty string, I would assume I can do other string operations on it, but that may fail if it is null. Does that matter here? Would it make sense to change this to something like
```suggestion
        Returns true if operand has no value.
```